plugins/usernames.py

```diff
@@ -56,6 +60,26 @@ async def annika(msg: Message) -> None:
     await msg.reply("enjoy ur italian joke punishment room")
 
 
+@command_wrapper(aliases=("anto",))
+async def antonio(msg: Message) -> None:
```

Comment:
```suggestion
@command_wrapper(aliases=("anto", "antonio"))
async def antonio200509(msg: Message) -> None:
```
We should always include the original nickname and use it for the function name.